reference/providers/kubernetes-v2/index.md

```diff
@@ -0,0 +1,116 @@
+are managed in Spinnaker today using other provider (including the Kubernetes
+Provider V1), so the below sub-sections will attempt to
+explain the differences.
+
+## No Restrictive Naming Policies
+
+In order to allow users to deploy existing manifests without rewriting them to
+adhere to the [Frigga naming convention](https://github.com/Netflix/frigga),
+resources relationships (between applications/clusters) are managed
+using [Kubernetes
+annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/).
+These are managed using Spinnaker's
+[Moniker](https://github.com/spinnaker/moniker) library, and the
+policies/strategies are configurable per-account. See the below [reserved
+annotations](#reserved-annotations) section for more detail.
+
+## Accomodating Level-based Deployments
+
+See the [Kubernetes API
+conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status)
+for a description of an edge- vs. level-based API.
+
+Prior providers in Spinnaker focused on tracking an operation that modified a
+cloud resource. For example, if a resize operation was run, Spinnaker
+would monitor that operation until the specific resize target was met. This
+runs counter to the Kubernetes level-based API, which only cares about the
+desired state of a resource being satisfied. In order to address this, we
+introduced the concept of __Manifest Stability__:
+
+A deployed manifest is consided __stable__ when the Kubernetes
+Controller-Manager no longer needs to modify it, and it is "ready". This
+definition is clearly very dependent on the manifest's kind (e.g. `Service` vs.
+`StatefulSet`), but for example, a `Deployment` is stable when its managed pods
+are updated, available, and ready (meaning they are running your desired
+container, and are serving traffic). Each kind's stability requirements are
+detailed in the [Resource Mapping](#resource-mapping) section.
+
+This concept of Manifest Stability is how Spinnaker ensures that operations
+have succeeded. Since there are a number of reasons why a manifest never
+becomes stable (lack of CPU quota, failing readiness checks, no IP for a
+service to bind...) every stage that modifies or deploys a manifest will wait
+until your affected manifests are stable, or timeout after a configurable
+period.
+
+## Deploying Manifests Stored Externally to Spinnaker
+
+To allow users to store & version their manifest definitions in Git (or
+elsewhere) outside of Spinnaker's pipeline store, we needed another way to
+supply manifest definitions to Pipeline executions.
+
+Using Spinnaker's new Artifact mechanism, we allow file modifications/creations
+to be surfaced as artifacts in pipeline executions. For example, you can now
+configure a pipeline that triggers either whenever 1. a new Docker image is
+uploaded, or 2. your manifest file is changed in git.
+
+# Reserved Annotations
```

Comment:
I know you mentioned this concept up above, but could still use an intro sentence or two here.